adafruit_sharpmemorydisplay.py

```diff
@@ -105,7 +116,7 @@ def show(self):
             image_buffer.extend(self._buf)
             spi.write(image_buffer)
 
-    def image(self, img):
+    def image(self, img: Image):
```

Comment:
`image()` can have a return type of None
```
def image(self, img: Image) -> None:
```

And similarly the `show()` method a few lines up from here in the code can have a `None` return type.

Reply:
Thanks so much for the tip!